src/com/videojs/providers/HLSProvider.as

```diff
@@ -254,13 +254,19 @@ package com.videojs.providers{
         public function get networkState():int {
           return _networkState;
         }
-
-        /**
-         * Should return the amount of media that has been buffered, in seconds, or 0 if
-         * this value is unknown or unable to be determined (due to lack of duration data, etc)
+        
+       /**
+         * Should return an array of normalized time ranges currently
+         * buffered of the media, in seconds.
          */
-        public function get buffered():Number {
-          return _bufferedTime;
+        public function get buffered():Array{
+            if(_bufferedTime) {
+                return [[
+                    _position - _bufferedTime,
```

Comment:
returned array does not seem correct, 
IMHO should be
[[ _position, _bufferedTime]]

if you want to return something even more accurate, taking into account the backbuffer, you can return

[[ _backBufferTime, _bufferedTime]]

you can compute _backBufferTime here https://github.com/mangui/video-js-swf/pull/9/files#diff-f497aef833a75fe7ea0dfa925fa2fbeeR105
with _backBufferTime = event.mediatime.position - event.mediatime.backbuffer;